.github/workflows/ci.yml

```diff
@@ -83,70 +69,54 @@ jobs:
         shell: bash
       - name: Set environment variables
         run: |
-          echo "Setting variables for '${{ matrix.p4est_test }}'..."
-          echo "P4EST_TEST=${{ matrix.p4est_test }}" >> $GITHUB_ENV
-          echo "P4EST_TEST=${{ matrix.p4est_test }}"
-          if [[ ${{ matrix.p4est_test }} == "P4EST_JLL_USES_MPI_PRE_GENERATED_BINDINGS" ]]; then
+          echo "Setting variables for '${{ matrix.JULIA_P4EST_TEST }}'..."
+          echo "JULIA_P4EST_TEST=${{ matrix.JULIA_P4EST_TEST }}" >> $GITHUB_ENV
+          echo "JULIA_P4EST_TEST=${{ matrix.JULIA_P4EST_TEST }}"
+          if [[ ${{ matrix.JULIA_P4EST_TEST }} == "P4EST_JLL_MPI_DEFAULT" ]]
+          then
             echo "(no additional environment variables required)"
-          elif [[ ${{ matrix.p4est_test }} == "P4EST_JLL_USES_MPI" ]]; then
-            echo "JULIA_P4EST_GENERATE_BINDINGS=yes" >> $GITHUB_ENV
-            echo "JULIA_P4EST_GENERATE_BINDINGS=yes"
-          elif [[ ${{ matrix.p4est_test }} == "P4EST_CUSTOM_NON_MPI" ]]; then
-            echo "JULIA_P4EST_GENERATE_BINDINGS=yes" >> $GITHUB_ENV
-            echo "JULIA_P4EST_GENERATE_BINDINGS=yes"
-            P4EST_TMP=`pwd`/libp4est_tmp_non_mpi
-            echo "P4EST_TMP=$P4EST_TMP" >> $GITHUB_ENV
-            echo "P4EST_TMP=$P4EST_TMP"
-            echo "JULIA_P4EST_LIBRARY=$P4EST_TMP/prefix/lib/libp4est.so" >> $GITHUB_ENV
-            echo "JULIA_P4EST_LIBRARY=$P4EST_TMP/prefix/lib/libp4est.so"
-            echo "JULIA_P4EST_INCLUDE=$P4EST_TMP/prefix/include" >> $GITHUB_ENV
-            echo "JULIA_P4EST_INCLUDE=$P4EST_TMP/prefix/include"
-          elif [[ ${{ matrix.p4est_test }} == "P4EST_CUSTOM_USES_MPI" ]]; then
-            echo "JULIA_P4EST_GENERATE_BINDINGS=yes" >> $GITHUB_ENV
-            echo "JULIA_P4EST_GENERATE_BINDINGS=yes"
-            P4EST_TMP=`pwd`/libp4est_tmp_uses_mpi
+          elif [[ ${{ matrix.JULIA_P4EST_TEST }} == "P4EST_CUSTOM_MPI_DEFAULT" ]] || [[ ${{ matrix.JULIA_P4EST_TEST }} == "P4EST_CUSTOM_MPI_CUSTOM" ]]
```

Comment:
I believe there is no `P4EST_CUSTOM_MPI_DEFAULT` test setup, is there? I find the variable in this file only in this line. Or is it used/set somewhere else? At least in lines 45ff. in the matrix setup it is not one of the values.

Reply:
That's right. I wanted to add something like this but realized that we would need to figure out where we can get the compilers for the MPI binaries used by MPI.jl from. 
Removed in 856902a.